src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -140,17 +140,17 @@ def renew_vae_attention_paths(old_list, n_shave_prefix_segments=0):
         new_item = new_item.replace("norm.weight", "group_norm.weight")
         new_item = new_item.replace("norm.bias", "group_norm.bias")
 
-        new_item = new_item.replace("q.weight", "query.weight")
-        new_item = new_item.replace("q.bias", "query.bias")
+        new_item = new_item.replace("q.weight", "to_q.weight")
+        new_item = new_item.replace("q.bias", "to_q.bias")
 
-        new_item = new_item.replace("k.weight", "key.weight")
-        new_item = new_item.replace("k.bias", "key.bias")
+        new_item = new_item.replace("k.weight", "to_k.weight")
+        new_item = new_item.replace("k.bias", "to_k.bias")
 
-        new_item = new_item.replace("v.weight", "value.weight")
-        new_item = new_item.replace("v.bias", "value.bias")
+        new_item = new_item.replace("v.weight", "to_v.weight")
+        new_item = new_item.replace("v.bias", "to_v.bias")
 
-        new_item = new_item.replace("proj_out.weight", "proj_attn.weight")
-        new_item = new_item.replace("proj_out.bias", "proj_attn.bias")
+        new_item = new_item.replace("proj_out.weight", "to_out.0.weight")
+        new_item = new_item.replace("proj_out.bias", "to_out.0.bias")
```

Comment:
the old and new names aren't fully qualified and the new names actually have the old names as a suffix in most of these. which means if the state dict already has the new names, this will further modify them to be incorrect.

re: `group_norm.weight` will become `group_group_norm.weight` and `to_q.weight` will become `to_to_q.weight`.

the strings to be replaced need to be complete in order to know that they can actually be replaced.

for instance 
```
new_item = new_item.replace("q.weight", "to_q.weight")
```
should probably be
```
new_item = new_item.replace(".q.weight", "to_q.weight")
```